### PR TITLE
feat(packages): Add NeuVector database and db generator

### PR DESCRIPTION
**Pull request by EyeCantCU**
Needed to generate database for NeuVector scanner

### CODE REVIEW

**Review by jamonation — DISMISSED** [review]

File: neuvector-dbgen.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          output: updater
+          prefix: usr/local
+          subpackage: true
+          vendor: true
+      - uses: strip
+
+test:
+  pipeline:
+    - runs: |
+        dbgen --help
```
Comment (jamonation) [inline neuvector-dbgen.yaml:51]: Looks like this --help is exiting non-zero :(

Comment (EyeCantCU) [reply neuvector-dbgen.yaml:51]: How on earth....

Comment (EyeCantCU) [reply neuvector-dbgen.yaml]: Updated the test to just check for the executable. While --help is built in, it still returns nonzero as if it wasn't 🤦. This is great

**Review by jamonation — DISMISSED** [review]

**Review by jonjohnsonjr — APPROVED** [review]

**Comment by xnox** [comment]
This seems to take over 3 hours to build in post-submit and is killing all builds.

I am proposing to revert this temporarily, check why it takes so long to build, and possibly resubmit it by itself - and/or split (it seems to try to download all CVE data of all linux distributions ever and parse it all and stalls).

**Pull request merged**